content/docs/components/serving/tritoninferenceserver.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "NVIDIA Triton Inference Server"
```

Comment:
Thanks! Since this PR deletes a page, you should also add a redirect to catch any incoming links from outside the doc set (blogs, GitHub, bookmarks, etc). You can add a redirect in this file:
https://github.com/kubeflow/website/blob/master/content/_redirects

Reply:
Added